### PR TITLE
add data hub

### PR DESCRIPTION
**Pull request by Cathy0908**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,11 @@ Please refer to the following model zoo for more details.
 - [classification model zoo](docs/source/model_zoo_cls.md)
 - [detection model zoo](docs/source/model_zoo_detection.md)
 
+## Data Hub
+
+EasyCV summarized various datasets in different fields.
```
Comment (wenmengzhou) [inline README.md:80]: EasyCV have collected dataset info for different senarios,  making it easy for users to fintune or evaluate models in EasyCV modelzoo

File: docs/source/data_hub.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+- [Classification data](#Classification-data)
+- [Object Detection](#Object-Detection)
+- [Self-Supervised Learning](#Self-Supervised-Learning)
+- [Pose](#Pose)
+- [Image Segmentation](#Image-Segmentation)
+
+## Classification data
+
+| Name                                                         | Field               | Describtion                                                  | Download                                                     | Support                                 |
```
Comment (wenmengzhou) [inline docs/source/data_hub.md:15]: Support -> Dataset API support

File: docs/source/data_hub.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+EasyCV summarized various datasets in different fields. At present, we support part of them, and we will gradually support remainings.
+
+**For datasets we already support, please refer to: [prepare_data.md](https://github.com/alibaba/EasyCV/blob/master/docs/source/prepare_data.md).**
+
+- [Classification data](#Classification-data)
+- [Object Detection](#Object-Detection)
+- [Self-Supervised Learning](#Self-Supervised-Learning)
+- [Pose](#Pose)
+- [Image Segmentation](#Image-Segmentation)
```
Comment (wenmengzhou) [inline docs/source/data_hub.md:11]: Self-Supervised Learning
Classification data
Object Detection
Image Segmentation -> semantic segmentation or instance segmentation?
Pose

Comment (wenmengzhou) [reply docs/source/data_hub.md:11]: follow the upper order

Comment (Cathy0908) [reply docs/source/data_hub.md]: Image Segmentation -> for all segmentation  senarios

File: README.md (hunk 1 above)
Comment (Cathy0908) [reply README.md]: done!

File: docs/source/data_hub.md (hunk 2 above)
Comment (Cathy0908) [reply docs/source/data_hub.md]: done!

File: docs/source/data_hub.md (hunk 3 above)
Comment (Cathy0908) [reply docs/source/data_hub.md]: done!

**Review by wenmengzhou — APPROVED** [review]

**Pull request merged**